src/app/workspace/resource/resource-instance-form/select-properties/switch-properties/switch-properties.component.html

```diff
@@ -0,0 +1,20 @@
+<span [ngSwitch]="property.objectType">
+    <dsp-text-value-as-string #createVal *ngSwitchCase="'ReadTextValueAsString'" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-text-value-as-string>
+    <dsp-text-value-as-xml #createVal *ngSwitchCase="'ReadTextValueAsXml'" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-text-value-as-xml>
+    <dsp-int-value #createVal *ngSwitchCase="constants.IntValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-int-value>
+    <dsp-boolean-value #createVal *ngSwitchCase="constants.BooleanValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-boolean-value>
+    <dsp-uri-value #createVal *ngSwitchCase="constants.UriValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-uri-value>
+    <dsp-decimal-value #createVal *ngSwitchCase="constants.DecimalValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-decimal-value>
+    <dsp-color-value #createVal *ngSwitchCase="constants.ColorValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-color-value>
+    <dsp-interval-value #createVal *ngSwitchCase="constants.IntervalValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-interval-value>
+    <dsp-time-value #createVal *ngSwitchCase="constants.TimeValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-time-value>
+    <dsp-geoname-value #createVal *ngSwitchCase="constants.GeonameValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-geoname-value>
+    <dsp-link-value #createVal *ngSwitchCase="constants.LinkValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"
+                [parentResource]="parentResource" [propIri]="property.id"></dsp-link-value>
+    <dsp-date-value #createVal *ngSwitchCase="constants.DateValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"></dsp-date-value>
+    <dsp-list-value #createVal *ngSwitchCase="constants.ListValue" [mode]="mode" [parentForm]="parentForm" [formName]="formName"
+                [propertyDef]="property"></dsp-list-value>
+    <span *ngSwitchDefault>
+        <p>Cannot match any value component.</p>
```

Comment:
Could you add the property.objectType here? So that we know, which value component is missing here?
e.g.

```html
    <span *ngSwitchDefault>
        <p>Cannot match any value component for <strong>{{property.objectType}}</strong></p>
    </span>
```

Reply:
added in 5f4bb1d